test/fixtures/keys/crypto-extensions.pem

```diff
@@ -0,0 +1,23 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
You should modify `test/fixtures/keys/Makefile` such that it can be used to generate this certificate, unless that is not feasible for some reason.